flink-yarn-tests/src/test/java/org/apache/flink/yarn/YarnTestBase.java

```diff
@@ -1070,25 +1059,18 @@ protected void runWithArgs(
             // this lets the test fail.
             throw new RuntimeException("Runner failed", runner.getRunnerError());
         }
-        Assert.assertTrue(
-                "During the timeout period of "
-                        + startTimeoutSeconds
-                        + " seconds the "
-                        + "expected string \""
-                        + terminateAfterString
-                        + "\" did not show up.",
```

Comment:
I think this message is still useful and shouldn't be lost.

Reply:
@zentol Added.